0.2.4 CaseSensitivePaths error

I upgraded from 0.2.3 from to 0.2.4, mainly to have the ability to set startDate and endDate to null, which I think will allow me to have the input field start empty. However, once I started the webpack dev server I got this error. 

```
ERROR in ./node_modules/vue2-daterange-picker/dist/lib/vue-daterange-picker.min.js
Module not found: Error: [CaseSensitivePathsPlugin] `/Users/tim/development/xxxx/node_modules/Vue/dist/vue.runtime.esm.js` does not match the corresponding path on disk `vue`.
```

--- a/build/webpack.lib.conf.js
+++ b/build/webpack.lib.conf.js
@@ -17,7 +17,7 @@ baseWebpackConfig.entry = {
 
 var webpackConfig = merge(baseWebpackConfig, {
   externals: {
-    vue: 'Vue',
+    vue: 'vue',
     moment: 'moment'
   },
   module: {

